mart/attack/adversary.py

```diff
@@ -179,15 +181,20 @@ def forward(self, *, batch: torch.Tensor | list | dict, model: Callable):
         self.attacker.fit_loop.max_epochs += 1
         self.attacker.fit(self, train_dataloaders=cycle([batch_and_model]))
 
-        # Get the input_adv for enforcer checking.
+    def forward(self, *, batch=None, input=None, target=None, enforce=True):
```

Comment:
You should be able to change this signature to `def forward(self, batch, enforce=True)`

Reply:
Fixed in [52d3b23](https://github.com/IntelLabs/MART/pull/215/commits/52d3b23174b5e34fb5b18639fb845477e939bae3)